src/node.rs

```diff
@@ -2,34 +2,36 @@ use core::ops::Drop;
 
 use crate::forest::Forest;
 use crate::geometry::Size;
-use crate::id::{self, NodeId};
+use crate::id::{Allocator, Id, NodeId};
 use crate::number::Number;
 use crate::result::Layout;
-use crate::style::*;
-use crate::sys;
+use crate::style::Style;
+#[cfg(feature = "std")]
+use crate::sys::Box;
```

Comment:
```suggestion
#[cfg(any(feature = "std", feature = "alloc"))]
use crate::sys::Box;
```
I think we only do not have this with no features enabled.

Reply:
This worked :)